app/src/main/java/com/voipgrid/vialer/logging/RemoteUncaughtExceptionHandler.java

```diff
@@ -17,10 +17,6 @@
         mDefaultHandler = Thread.getDefaultUncaughtExceptionHandler();
     }
 
-    public RemoteUncaughtExceptionHandler(RemoteLoggingActivity remoteLoggingActivity) {
```

Comment:
I put it in as a comment last time as it didn't apply to any specific line so you probably didn't notice it, but:

> You should also be able to remove the 'spindle-lock-ring' line from the gradle file